openapi_python_client/schema/openapi_schema_pydantic/components.py

```diff
@@ -44,7 +45,11 @@ class Components(BaseModel):
     links: Optional[Dict[str, Union[Link, Reference]]] = None
     """An object to hold reusable [Link Objects](#linkObject)."""
 
+    callbacks: Optional[Dict[str, Union[Callback, Reference]]] = None
+    """An object to hold reusable [Callback Objects](#callbackObject)."""
+
     class Config:
+        extra = Extra.forbid
```

Comment:
I'm not sure forbid is the right call here for spec compliance. Doesn't OpenAPI allow for extensions that can basically add whatever they want?

Also not sure it gains us anything but definitely causes a breaking change, so might be better to leave it out.

Reply:
It does indeed: https://swagger.io/docs/specification/openapi-extensions/ ; resolved on https://github.com/triaxtec/openapi-python-client/pull/426/commits/ac61bdaee1c8409dbd218140289252c966ece656

Reply:
Also, I still thinks that these changes should be considered as breaking changes, as exemple: 
```
-   tokenUrl: Optional[str] = None
+   tokenUrl: Optional[AnyUrl] = None
 ```
 
 Previously an invalid url was accepted here and passed validation, but will not anymore

Reply:
Oh yup, agreed that there's still some potential for breakage.